src/components/HeroSection/HeroSection.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Link } from "react-router-dom";
+import "./HeroSection.css";
+const HeroSection = () => {
+    return (
+        <header className='hero-wrapper flex-column flex-center'>
+            <img src="./taking-notes-hero.svg" alt="Taking Notes through UNOTE" className='hero-image responsive-img' />
```

Comment:
provide alt attribute to img tag

Reply:
Ok alt value is incorrect will correct it!!